crates/epaint/src/tessellator.rs

```diff
@@ -1783,11 +1783,13 @@ impl Tessellator {
                 continue;
             }
 
+            let final_pos = galley_pos + row.pos.to_vec2();
```

Comment:
maybe call this `final_row_pos` for extra clarity

Reply:
Done.